quilt/README.md

```diff
@@ -0,0 +1,241 @@
+# Running Ray with Quilt
+
+Quilt is a way to deploy and network containers, meaning that Quilt will boot
+machines and set up containers on supported cloud providers (AWS, GCE, Azure)
+for you. Learn more about this project [here](http://quilt.io).
+
+## Introduction
+Quilt uses a domain specific language, Stitch (files that end with `.spec`, to 
+specify distributed applications, independent of the specific infrastructure
+they run on. This folder contains [`ray.spec`](ray.spec) and 
+[`ray-runner.spec`](ray-runner.spec). `ray.spec` provides abstractions for
+creating ray containers - the function `ray.New` creates `nWorker` worker nodes,
+and one head node, all prefixed with `prefix`.
+
+`ray-runner.spec` defines the infrastructure When you run `ray-runner.spec`, Quilt will
+
+## Step 1: Installing Quilt
```

Comment:
I'm not entirely sure it's appropriate for us to have detailed instructions on installing quilt here.  Would make much more since to just say something like "This assumes you have quilt installed. Go here to set it up."  That way as our install instructions change (i.e. we release binaries), we don't have to update each user.